vllm/transformers_utils/tokenizer_group/tokenizer_group.py

```diff
@@ -55,19 +55,31 @@ def _raise_if_input_too_long(self,
     def encode(self,
                prompt: str,
                request_id: Optional[str] = None,
-               lora_request: Optional[LoRARequest] = None) -> List[int]:
+               lora_request: Optional[LoRARequest] = None,
+               add_special_tokens: Optional[bool] = None) -> List[int]:
         tokenizer = self.get_lora_tokenizer(lora_request)
-        ret = tokenizer.encode(prompt)
+        if add_special_tokens is not None:
+            ret = tokenizer.encode(  # type: ignore[call-arg]
+                prompt,
+                add_special_tokens=add_special_tokens)
```

Comment:
Let's move `vllm.multmodal.processor._encode` to `vllm.transformers_utils` so we can reuse the logic and avoid this type error.